docs/installation/quick-start.md

```diff
@@ -7,6 +7,11 @@ Bee is packaged for Ubuntu, Raspbian, Debian and CentOS based Linux distribution
 
 If your system is not supported, please see the [manual installation](/docs/installation/manual) section for information on how to install Bee.
 
+The overview of the installation process:
+ 1. set up an external signer for Bee (e.g. install *bee-clef*)
+ 2. install the Bee application package
+ 3. fund your node with ETH and BZZ
```

Comment:
g(ETH) and g(BZZ) for the time being. Is this worthy to add according to you?

Reply:
@Eknir i think we should write the docs in a future-proof way, and only point out that it's running on the testnet currently at the sections where it's relevant, e.g. where funding itself is described.

what do you think?

Reply:
@Eknir or maybe in a big fat warning at the opening page of the docs?

Reply:
Leaving that up to @significance . Anyways. Good to go for me as is.

Reply:
Yep agree with @Eknir i think we should keep it the same as the rest of the docs for now to avoid to confusion, i think we can change them en-masse when we go live.

```suggestion
 3. fund your node with gETH and gBZZ
```